src/components/activity-page/embeddable.test.tsx

```diff
@@ -6,30 +6,52 @@ import { PageLayouts } from "../../utilities/activity-utils";
 import { mount, ReactWrapper } from "enzyme";
 import { EmbeddableWrapper } from "../../types";
 import { act } from "react-dom/test-utils";
-import { DefaultManagedInteractive } from "../../test-utils/model-for-tests";
+import { DefaultManagedInteractive, DefaultXhtmlComponent } from "../../test-utils/model-for-tests";
 
 describe("Embeddable component", () => {
-  it("renders a text component", async () => {
+  it("renders a non-callout text component", async () => {
     const embeddableWrapper: EmbeddableWrapper = {
       "embeddable": {
+        ...DefaultXhtmlComponent,
         "content": "<p><strong>This is a page with full width layout</strong>.&nbsp;&nbsp;Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>",
-        "is_full_width": true,
-        "is_hidden": false,
-        "name": "",
-        "type": "Embeddable::Xhtml",
-        "ref_id": "271776-Embeddable::Xhtml"
+        "is_callout": false,
+        "is_full_width": true
       },
       "section": "header_block"
     };
 
     let wrapper: ReactWrapper;
     await act(async () => {
-      wrapper = mount(<Embeddable embeddableWrapper={embeddableWrapper} isPageIntroduction={false} questionNumber={1} pageLayout={PageLayouts.Responsive}/>);
+      wrapper = mount(<Embeddable embeddableWrapper={embeddableWrapper} questionNumber={1} pageLayout={PageLayouts.Responsive}/>);
       expect(wrapper.containsMatchingElement(<div>Loading...</div>)).toEqual(true);
     });
 
     await waitFor(() => {
       expect(wrapper.text()).toContain("This is a page");
+      expect(wrapper.html()).toContain('class="textbox');
+      expect(wrapper.html()).not.toContain('class="textbox callout"');
```

Comment:
As I said in slack, the failure of these tests because of an additional space points out the brittleness of testing for exact class strings. As you encountered here, adding a space or changing the order of the classes breaks the tests but obviously wouldn’t affect the functionality. It would be preferable to switch to something like [hasClass()](https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/hasClass.html), but I’d also understand if you didn’t want to bother at this point.